encoding.h

```diff
@@ -154,17 +154,34 @@
 #define MIP_MEIP            (1 << IRQ_M_EXT)
 #define MIP_SGEIP           (1 << IRQ_S_GEXT)
 #define MIP_LCOFIP          (1 << IRQ_LCOF)
+#define MIP_MSDEIP          (1 << IRQ_MSDEI)
 #define MIP_RAS_LOW_PRIO    (1ULL << IRQ_RAS_LOW_PRIO)
 #define MIP_RAS_HIGH_PRIO   (1ULL << IRQ_RAS_HIGH_PRIO)
 
 #define MIP_S_MASK          (MIP_SSIP | MIP_STIP | MIP_SEIP)
 #define MIP_VS_MASK         (MIP_VSSIP | MIP_VSTIP | MIP_VSEIP)
 #define MIP_HS_MASK         (MIP_VS_MASK | MIP_SGEIP)
 
+#define MIE_SSIE            (1 << IRQ_S_SOFT)
+#define MIE_VSSIE           (1 << IRQ_VS_SOFT)
+#define MIE_MSIE            (1 << IRQ_M_SOFT)
+#define MIE_STIE            (1 << IRQ_S_TIMER)
+#define MIE_VSTIE           (1 << IRQ_VS_TIMER)
+#define MIE_MTIE            (1 << IRQ_M_TIMER)
+#define MIE_SEIE            (1 << IRQ_S_EXT)
+#define MIE_VSEIE           (1 << IRQ_VS_EXT)
+#define MIE_MEIE            (1 << IRQ_M_EXT)
+#define MIE_SGEIE           (1 << IRQ_S_GEXT)
+#define MIE_LCOFIE          (1 << IRQ_LCOF)
+#define MIE_MSDEIE          (1 << IRQ_MSDEI)
+
```

Comment:
Let's get rid of this.  It is a property of the ISA that MIP and MIE bits line up with each other, so having a redundant set of macros is just more code to maintain.